sched/adaptdl_sched/policy/pollux.py

```diff
@@ -341,6 +360,10 @@ def _repair(self, pop, **kwargs):
         with np.errstate(divide="ignore", invalid="ignore"):
             states = np.amin(np.floor_divide(states, job_resources),
                              where=job_resources > 0, initial=99, axis=-1)
+        # Only choose solutions which have at least min_replicas allocations
+        min_replicas = np.array([j.min_replicas for j in self._jobs])
+        mask = np.sum(states, axis=-1) < min_replicas
+        states[mask] = 0
```

Comment:
How well does this scale in terms of `min_replicas`? On average, each job in the GA state suffers an average of 1-2 mutations. Resetting back to 0 each generation means a job will need to suffer at least `min_replicas` mutations in a single generation to pass this condition. Seems like if `min_replicas` is larger than 5-6 replicas then it's very unlikely this will happen.

Reply:
During `_muation`, we assign minimum allowable replicas for the job to the `states` to produce viable stream of population which can then be constrained in `_repair`. This has shown good scalability experimentally.